include/uxr/client/config.h.in

```diff
@@ -29,6 +29,9 @@
 #cmakedefine PLATFORM_NAME_LINUX
 #cmakedefine PLATFORM_NAME_WINDOWS
 #cmakedefine PLATFORM_NAME_NUTTX
+#cmakedefine PLATFORM_EXTERNAL
+
+#cmakedefine PLATFORM_EXTERNAL_TRANSPORT_HEADER_FILENAME <uxr/client/profile/transport/serial/@PLATFORM_EXTERNAL_TRANSPORT_HEADER_FILENAME@>
```

Comment:
Maybe the `PLATFORM_` prefix is not necessary, what do you think?

Reply:
`PLATFORM_` prefix removed.